### PR TITLE
SD card datalogger support

### PR DESCRIPTION
**Pull request by cmastudios**

### CODE REVIEW

**Comment by cmastudios** [comment]
Performance test results: writing 4MB through FatFs in 4KB chunks took 22s, for a throughput of 200kB/s

File: 2020/Controllers/Shared/Drivers/drv_sd.c [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+	}
+	if (resp == 0xff)
+	{
+		return RES_NOTRDY;
+	}
+
+	for (UINT j = 0; j < count; j++)
+	{
+//		// Send one 0xff byte to prep for sending data block
+//		drv_spi_transfer(DRV_SPI_CHANNEL_SD, 0xff);
```
Comment (ethanshry) [inline 2020/Controllers/Shared/Drivers/drv_sd.c:237]: why comment way outta line tho

File: 2020/Controllers/Shared/Drivers/drv_rtc.c [hunk 2]
```diff
@@ -0,0 +1,78 @@
+#include "drv_rtc.h"
```
Comment (ethanshry) [inline 2020/Controllers/Shared/Drivers/drv_rtc.c:1]: Fancy header?
```
/* 
 * File:   drv_rtc.h
 * Author: Connor
 *
 * Created on March 7, 2021, 11:57 AM
 */
```

File: 2020/Controllers/Shared/Drivers/drv_rtc.h [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* 
+ * File:   drv_rtc.h
+ * Author: Connor
+ *
+ * Created on March 7, 2021, 11:57 AM
+ */
```
Comment (ethanshry) [inline 2020/Controllers/Shared/Drivers/drv_rtc.h:6]: I also tend to like file descriptions here (i.e. broadly what is in this file) but maybe is unnecesarry

**Review by ethanshry — APPROVED** [review]

**Pull request merged**